### PR TITLE
feat(user):

### PR DESCRIPTION
**Pull request by joeeasy**
- users can view their names

[finishes 12232123232]

### CODE REVIEW

**Comment by Frediflexta** [comment]
You used ES5!!

File: index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const user = 'joeeasy';
+
```
Comment (ajimae) [inline index.js]: Nice work but why the blank line? can we remove the blank lines

File: index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+const getUser = (username) => console.log('username :', username);
+
```
Comment (anasey01) [inline index.js]: why are there white spaces?

File: index.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+getUser(user);
```
Comment (uwaelpis) [inline index.js]: Great job

Comment (Frediflexta) [inline index.js]: You mean you used ES5

**Pull request merged**